ui/.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 const neutrino = require('neutrino');
 
-module.exports = neutrino().eslintrc();
+module.exports = neutrino().eslintrc();
```

Comment:
Could you revert the changes to this file?

Reply:
I changed the file accidently in a commit :sweat_smile: , I'll revert the changes